Configurateur/front/lib/screens/user-list/user_list.dart

```diff
@@ -105,7 +105,7 @@ class _UserPageState extends State<UserPage> {
 
   Future<void> fetchUserMobile() async {
     final response = await http
-        .get(Uri.parse('http://${serverIp}:8080/api/dev/user/listall'));
+        .get(Uri.parse('http://${serverIp}:3000/api/mobile/user/listAll'));
```

Comment:
the server port should be add to serverIp, would be easier in case of change

Reply:
Gonna be done soon, for delivery